WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -93,6 +93,13 @@ final class OrderDetailsViewController: UIViewController {
         }
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        // scroll table view to top so that the next time the split view
+        // shows the detail screen, the table view content is at the top position
+        tableView.contentOffset = .zero
+        super.viewWillDisappear(animated)
+    }
```

Comment:
I was reusing the order details screen for every selected order, so I wanted to scroll the details view to the top every time a new item is selected, so that it looks better in collapsed mode.

However, I haven't yet found a good way to scroll the table view to top successfully, so I switched to creating a new details view for every item in 47d7489ea650f57e5443c9821b458e0e71efa684. This will keep the old behavior that we had with the navigation stack.

> Plus, you can notice that the selected order in the list, is no more selected after switching to a different tab.

This is actually a different issue that I tried to remedy in 3132df8a4de80b33efac478d545ba73d14c9224f. I see that _sometimes_ the highlight state is still lost and I need to look into this further.